inst/rmarkdown/templates/oup/resources/template.tex

```diff
@@ -27,15 +50,14 @@
 }
 
 
-% For knitr::kable functionality
+% For knitr::kable and kableExtra functionality
 \usepackage{booktabs}
 \usepackage{longtable}
-
-%% To allow better options for figure placement
-%\usepackage{float}
-
-% Packages that are supposedly required by OUP sty file
-\usepackage{amssymb, amsmath, geometry, amsfonts, verbatim, endnotes, setspace}
+\usepackage{float}
+\usepackage{colortbl}
+\usepackage{pdflscape}
+\usepackage{tabu}
+\usepackage{threeparttable}
 
 % For code highlighting I think
```

Comment:
I think we need this instead: 
````
$if(highlighting-macros)$
$highlighting-macros$
$endif$
```` 

See https://github.com/rstudio/rticles/pull/419#discussion_r678165258

Reply:
Yes. I copied this from somewhere before understanding how `$highlighting-macros$` works.

Reply:
Now fixed.